### DISCUSSION TITLE
I can't purchase a subscription

### DISCUSSION THREAD

**Discussion by NeuralAIM** [community/community#19093] [Copilot Conversations] [answered]
**I tried to buy a subscription, but after that I got such a sign:**

![image](https://user-images.githubusercontent.com/75082388/174894292-d75053a4-694c-4a45-8e44-fabd0de38a2d.png)

_I know that OpenAI is closed to my country, and that's why my account was deleted when they found out I was from Russia. Is it possible that Copilot has the same policy towards Russia, or is it because of the current political situation?_

---

**It is strange that I was not informed that the service is not available for my country.**

_For example, the way OpenAI does:_

![image](https://user-images.githubusercontent.com/75082388/174895157-a95d0766-b41d-4ad3-9c24-e7ad285e2cc4.png)

When paying, there is Russia in the lists of countries, so I thought I could purchase a subscription directly. If this is not possible, can I ask my friend to pay for my subscription? (will my account be blocked because of this, etc.?)

**Comment by melvincwng** [comment]
Hi @NeuralAIM, for account-related issues, please kindly raise a ticket and contact [Github Support](https://support.github.com/) over here.

They would usually respond within a few working days.

Hope this helps, thanks

**Reply by melvincwng** [reply]
@NeuralAIM, if you find this reply helpful, please 'Mark as answer' so that others are aware of the solution, should they encounter the same issue in the future.

Thank you for your cooperation 👍🏻

**Comment by NeuralAIM** [comment, marked as answer]
I was born in Russia, so I can't buy or even try a 60-day trial subscription 😔

**Reply by Eric0052** [reply]
Sad，，，

**Comment by ohaui** [comment]
@NeuralAIM Hello! You can actually try it for free by pressing this button : ) Russians credit cards are banned from buying anything outside of Russia
![ghcplt](https://user-images.githubusercontent.com/42948516/175306099-8f9b5e30-d76e-4ac3-bb55-3450fff52d0d.png)

**Reply by NeuralAIM** [reply]
It's not about the card, but the fact that I live in Russia. And I was not given the opportunity to try a free test for 60 days, perhaps because at the moment I am not a paying customer:
![image](https://user-images.githubusercontent.com/75082388/175378319-f2c21004-8c68-4540-8582-de52e8a99588.png)

**Reply by NeuralAIM** [reply]
> It's not about the card, but the fact that I live in Russia. And I was not given the opportunity to try a free test for 60 days, perhaps because at the moment I am not a paying customer: ![image](https://user-images.githubusercontent.com/75082388/175378319-f2c21004-8c68-4540-8582-de52e8a99588.png)

In fact , more than 48 hours have already passed since the appearance of this sign :)

**Comment by thunder-007** [comment]
try using vpn

**Reply by NeuralAIM** [reply]
It won't help me anyway. I will either be blocked from paying, or the card will not be accepted. At the same time, I did not receive the claimed 60 days of the free period.

**Comment by NeuralAIM** [comment]
Really? Is it available to all developers? Haha)

![image](https://user-images.githubusercontent.com/75082388/175380275-e696d4dc-4bd6-4475-aabd-4cf4e938e1eb.png)
